### DISCUSSION TITLE
loops and stems clutter up library

### DISCUSSION THREAD

**Discussion by jhurlbut** [okio-ai/nendo-platform#8] [General]
The loops and stems of tracks get added to the library as separate tracks which makes it hard to navigate the library. It would be cool if there was some UX for them to be hidden away underneath the original track somehow.

**Comment by samim23** [comment]
Check the pull down selector next to the search which say "All type". There you can select "see just tracks" etc.

Check the detail view of a track, there you can see a "related track" section underneath, which would show you stems, etc

**Comment by samim23** [comment]
@jhurlbut do you feel, the default "show type" should be "tracks only"?
